src/coreclr/jit/redundantbranchopts.cpp

```diff
@@ -1619,11 +1618,13 @@ bool Compiler::optReachable(BasicBlock* const fromBlock, BasicBlock* const toBlo
                 return true;
             }
 
-            if ((succ->bbFlags & BBF_VISITED) != 0)
+            if (BitVecOps::IsMember(optReachableBitVecTraits, optReachableBitVec, succ->bbNum))
```

Comment:
We have this exact code in `fgRemoveDeadBlocks()`. Can we unify them?

Reply:
That version uses `BlockSet`, which I'm avoiding here because it is epoch sensitive.

In general, it would be nice to have the ability to represent (sparse) block sets that are independent of `bbNum` and not fixed capacity so that they can persist for longer stretches or be reused as general scratch sets, but we don't have anything like that yet.